common/dynamicconfig/constants.go

```diff
@@ -761,6 +761,11 @@ This config is EXPERIMENTAL and may be changed or removed in a later release.`,
 		0.5,
 		`HistoryHostErrorPercentage is the percentage of hosts that are unhealthy`,
 	)
+	HistoryHostStartingPercentage = NewGlobalFloatSetting(
+		"frontend.historyHostStartingPercentage",
+		0.05,
```

Comment:
nitpicky, but: "percentage" implies a number from 0 to 100. A fraction like 0.05 should be a "fraction".

Reply:
technically the correct word would be [proportion](https://math.stackexchange.com/questions/1350980/correct-term-for-percentage-in-decimal-form) according to the math stack exchange 😅 It looks like we use this notation frequently across the dynamic config. Maybe I'm wrong, but when I see a decimal and the unit is percentage, I assume this notation.

Reply:
I said it was nitpicky :)

"proportion" is also great

Reply:
changed!